internal/action/sync.go

```diff
@@ -85,6 +85,7 @@ func (s *Action) autoSync(ctx context.Context) error {
 	debug.Log("autosync - interval: %s", syncInterval)
 
 	if time.Since(ls) > syncInterval {
+		ctx = ctxutil.WithInteractive(ctx, true)
```

Comment:
It looks like you're relying on the interactive context key in the end, instead of the newly added autosync one, no?

Reply:
You might mean to use 
```suggestion
		ctx = ctxutil.WithAutoSync(ctx, true)
```
there, no?

Reply:
Yes, I intended to use the new context.